src/ggml-backend-impl.h

```diff
@@ -76,6 +76,8 @@ extern "C" {
     typedef void * ggml_backend_context_t;
 
     struct ggml_backend_i {
+        ggml_guid_t (*GGML_CALL get_guid)();
+
```

Comment:
Since the UUID is already stored in the `struct ggml_backend`, I don't think this needs to be part of the interface. Either on the interface or on the struct is fine, but putting it on both is redundant.

I cannot see any reason to change the UUID at run time, so I think that it would be better to just keep it as a constant on the struct, to be set by the backends during initialization in the `ggml_backend_xxx_init` functions.

Reply:
Alright I removed the interface function and its helper